recipes/fandango/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fandango" %}
+{% set version = "15.10.3" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/fandango-{{ version }}.tar.gz
+  sha256: a3a6fabe301c33d1715f267733b3a5e677de03d9b1e43e0111e7630402ecbeaf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools    
+    - future    
+  run:
+    - python >={{ python_min }}
+    - future
```

Comment:
This matches what you have in your setup.py but it seems some dependencies are missing.
At least `pytango` should be defined here.

```
import: 'fandango'
Unable to import fandango.*tango modules
```

Just a question: shouldn't that raise an error instead of just printing some message?
The import test passes but that sounds incorrect. I guess you can't do much without tango installed.

Reply:
Hi Benjamin, this is intentional

We do use fandango in some systems where the tango control system is not available, in extraction apis from archiving or web clients.

We just print the warning but allow to use the rest of functionality that does not depend on pytango

Reply:
I will modify the scripts to be entry points and then proceed with the merge

I'll make a new release for fandango and update yaml